.claude/CLAUDE.md

```
@@ -0,0 +1,326 @@
+### Testing Best Practices
+- Don't use export or module.exports in test files.
+- Don't use focused tests.
+- Make sure the assertion function, like expect, is placed inside an it() function call.
+- Don't use disabled tests.
+
+## Common Tasks
+- `npx ultracite init` - Initialize Ultracite in your project
+- `npx ultracite fix` - Format and fix code automatically
+- `npx ultracite check` - Check for issues without fixing
+
+## Example: Error Handling
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)